MediaBrowser.Model/Cryptography/PasswordHash.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MediaBrowser.Model.Cryptography
+{
+    public class PasswordHash
+    {
+        //Defined from this hash storage spec
+        //https://github.com/P-H-C/phc-string-format/blob/master/phc-sf-spec.md
+        //$<id>[$<param>=<value>(,<param>=<value>)*][$<salt>[$<hash>]]
+
+        public string Id;
+        public Dictionary<string, string> Parameters = new Dictionary<string, string>();
+        public string Salt;
+        public byte[] SaltBytes;
+        public string Hash;
+        public byte[] HashBytes;

```

Comment:
Change the public fields into properties.

Reply:
sure. I went through and updated everything in the class to use the privates as well.